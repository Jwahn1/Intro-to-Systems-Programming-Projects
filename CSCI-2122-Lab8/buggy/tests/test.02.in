new
append alpha
find alpha
insert gamma
find alpha
insert beta
end
