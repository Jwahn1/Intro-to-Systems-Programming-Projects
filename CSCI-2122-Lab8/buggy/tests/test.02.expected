[ ]
[ alpha ]
'alpha' found in list
[ alpha gamma ]
'alpha' found in list
[ alpha beta gamma ]
